DXRMapFixups/DeusEx/Classes/DXRFixupParis.uc

```diff
@@ -232,6 +233,12 @@ function AnyEntryMapFixes()
         break;
     case "10_PARIS_CHATEAU":
         FixConversationAddNote(GetConversation('NicoletteInStudy'),"I used to use that computer whenever I was at home");
+        foreach AllActors(class'FlagTrigger', ft) {
```

Comment:
Changing the flag triggers doesn't need to be done on AnyEntry, only first entry.

That being said, I'd probably rather see FixConversationFlag used here instead to change the flag referenced in the conversation from ChateauInCeller to ChateauInCellar